### PR TITLE
docs(rxBreadcrumbs): FRMW-164-Migrate rxBreadcrumbs to API Docs

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-164

- [x] Dev
- [x] Dev

Migrate `rxBreadcrumbs` to API Documentation.

### CODE REVIEW

**Comment by 100stacks** [comment]
screen shot 2015-09-22 at 3 06 13 pm
screen shot 2015-09-22 at 3 05 50 pm

**Comment by 100stacks** [comment]
screen shot 2015-09-22 at 3 05 50 pm

**Comment by bry4n** [comment]
LGTM

File: src/rxBreadcrumbs/rxBreadcrumbs.js [hunk 1]
```diff
@@ -4,7 +4,28 @@
...
+ *
+ * ## `rxBreadcrumbs` Available Options
+ * 
+ * A number of options are available:
+ *  
+ * - By default, the first breadcrumb will always have an URL of `'/'` and a name of `'Home'`. This can be changed
+ * with the `rxBreadcrumbsSvc.setHome()` method (see {@link rxBreadcrumbs.service:rxBreadcrumbsSvc rxBreadcrumbsSvc}).
+ *  
+ * - The argument `status` is also an optional argument to breadcrumbs. This leverages the tags defined in 
+ * [rxApp](/#/components/rxApp) to display status tags directly inside of breadcrumbs. The demo shows the use of the
```
Comment (CITguy) [inline src/rxBreadcrumbs/rxBreadcrumbs.js]: This link needs updated to `{@link rxApp rxApp}` to point to the `rxApp` API documentation.

File: src/rxBreadcrumbs/rxBreadcrumbs.js [hunk 2]
```diff
@@ -16,7 +37,15 @@ angular.module('encore.ui.rxBreadcrumbs', ['ngSanitize'])
 /**
  * @ngdoc service
  * @name rxBreadcrumbs.service:rxBreadcrumbsSvc
- * @description [TBD]
+ * @description
+ * By default, the first breadcrumb will always have an URL of `'/'` and a name of `'Home'`.  This can be changed
+ * with the `rxBreadcrumbsSvc.setHome()` method. 
+ * 
+ * ## `rxBreadcrumbsSvc.setHome()` Method
```
Comment (CITguy) [inline src/rxBreadcrumbs/rxBreadcrumbs.js]: Please, define this documentation for `breadcrumbService.setHome()` further down the file.

Comment (100stacks) [inline src/rxBreadcrumbs/rxBreadcrumbs.js]: By *further down the file*....do you want it just above?  Thanks.
```js
breadcrumbsService.setHome = function (path, name) {
         breadcrumbs[0] = {
             path: path,
             name: name || breadcrumbs[0].name
         };
     };
```

Comment (CITguy) [inline src/rxBreadcrumbs/rxBreadcrumbs.js]: Yes, that is correct.

**Comment by 100stacks** [comment]
Thanks @CITguy for the help with ngdocs!  It is fairly useful after the syntax explanation.

**Comment by CITguy** [comment]
LGTM.

@100stacks, please squash

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+  * FRMW-158 - Migrate rxActiveUrl docs in README to ngdocs ([b15af7ef](git@github.com:rackerlabs/encore-ui/commit/b15af7efba368f077131965eb1c59f45eb0a4c0e))
+  * FRMW-153 - Migrate rxActionMenu docs in README to ngdocs ([5a7f3f6a](git@github.com:rackerlabs/encore-ui/commit/5a7f3f6aab92436426817b71e14a2afd2b65dd01))
+  * FRMW-150 - Migrate configs API Docs in README to ngdocs ([ce100881](git@github.com:rackerlabs/encore-ui/commit/ce100881fbf8a729ebae9bf112f44849e16ef644))
+  * Create Typography style page ([4125d4ba](git@github.com:rackerlabs/encore-ui/commit/4125d4bac6c19315da5c60b91471aa9a3e688980))
+  * FRMW-156 - Migrate rxAccountInfo docs in README to ngdocs ([3ff14f61](git@github.com:rackerlabs/encore-ui/commit/3ff14f618a32691782e66b93f03d21e32a402a88))
+  * FRMW-153 - Migrate layout API Docs in README to ngdocs ([c78e098b](git@github.com:rackerlabs/encore-ui/commit/c78e098b4cca4e4df08ef9dac026541339f7c966))
+  * FRMW-151 - Migrate hotkey s API Docs in README to ngdocs ([a5477797](git@github.com:rackerlabs/encore-ui/commit/a5477797cfb206466672d6f7f0de91bb639a73db))
+
+
```
Comment (CITguy) [inline CHANGELOG.md]: @100stacks, please revert this file change.

**Comment by CITguy** [comment]
Something strange happened with your squash. I think you may have squashed too much in your history.

**Comment by CITguy** [comment]
Added "DO NOT MERGE" because there is some strangeness with the commit.

**Pull request merged**